dapr/proto/runtime/v1/dapr.proto

```diff
@@ -226,4 +229,15 @@ message ExecuteStateTransactionRequest {
 
   // The metadata used for transactional operations.
   map<string,string> metadata = 3;
-}
\ No newline at end of file
+}
+
+// RegisterActorTimerRequest is the message to register a timer for an actor of a given type and id.
+message RegisterActorTimerRequest {
+  string actor_type = 1;
+  string actor_id = 2;
+  string name = 3;
+  string due_time = 4;

```

Comment:
Can we use https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp instead?